homeassistant/components/whirlpool/sensor.py

```diff
@@ -107,14 +106,12 @@ class WhirlpoolSensorEntityDescription(
         key="state",
         name="State",
         icon=ICON_W,
```

Comment:
I think we can also move the icon attribute and just have a check in the entity as it's always one or the other and that isn't related to which description is used.